challenge-two/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "forta-challenge-2",
+  "displayName": "Nethermind Uniswap Forta Bot",
+  "version": "0.0.1",
+  "description": "Forta Bot that captures creation and update of bots deployed by Nethermind",
+  "longDescription": "This Forta Bot searches for transactions to the Forta Bot Registry that are sent from Nethermind's deployment address. The transactions are further filtered to provide alerts in the cases that a new bot is deployed, or a pre-existing bot is updated.",
+  "repository": "https://github.com/forta-network/forta-bot-sdk/tree/master/starter-project",
+  "licenseUrl": "https://github.com/forta-network/forta-bot-sdk/blob/master/starter-project/LICENSE.md",
+  "promoUrl": "https://forta.org",
+  "chainIds": [
+    137
+  ],
```

Comment:
Quick reminder that `"chainIds"` should contain all of the chain ids of the chains your bot is supposed to be monitoring.